src/vs/editor/common/viewModel/splitLinesCollection.ts

```diff
@@ -1451,6 +1472,10 @@ export class SplitLine implements ISplitLine {
 
 		return outputPosition;
 	}
+
+	public getInjectedTextAt(outputLineIndex: number, outputColumn: number): InjectedText | null {
+		return this._lineBreakData.getInjectedText(outputLineIndex, outputColumn);
```

Comment:
I think here we should pass in `outputColumn - 1`.

Reply:
Good catch. It will probably not be the last time I do this mistake 😉